mmv1/templates/terraform/custom_flatten/compute_router_range.go.erb

```diff
@@ -12,7 +12,7 @@
 	# See the License for the specific language governing permissions and
 	# limitations under the License.
 -%>
-func flatten<%= prefix -%><%= titlelize_range(range) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+func flatten<%= prefix -%><%= titlelize_property(range) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
```

Comment:
I think this is where the problem is - the `range` argument passed into the titlelize_property Ruby function. I believe it should be `titlelize_property(property)`

Reply:
This is from looking at other custom_flatten files like [/mmv1/templates/terraform/custom_flatten/alloydb_cluster_input_user_flatten.go.erb](https://github.com/GoogleCloudPlatform/magic-modules/blob/0b5518b2d4fb333ea8f09c88cad09f1be8022273/mmv1/templates/terraform/custom_flatten/alloydb_cluster_input_user_flatten.go.erb#L15C31-L15C59)